app/controllers/api/v2/mail_notifications_controller.rb

```diff
@@ -21,6 +30,84 @@ def index
 
       def show
       end
+
+      api :POST, "/users/:user_id/mail_notifications", N_("Create an email notification for a user")
```

Comment:
@yifatmakias 
Just semantic nitpick, we are not really creating a new notification, just adding a notification to the user. 
I would prefer if you would change the description to " Add an email notification to the user"